apps/vr-tests-react-components/package.json

```diff
@@ -27,6 +27,7 @@
     "@fluentui/react-card": "^9.0.4",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 26 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27351&commitId=e26f4e2e9de3ab5714d458d697a029e1e7f93891&env=prod)

✅ There was 26 screenshots added, 0 screenshots removed, 1903 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day - Dark Mode.chromium.png|0| Added|
|DatePicker Compat.DateRange- day - High Contrast.chromium.png|0| Added|
|DatePicker Compat.DateRange- day.chromium.png|0| Added|
|DatePicker Compat.DateRange- month - Dark Mode.chromium.png|0| Added|
|DatePicker Compat.DateRange- month - High Contrast.chromium.png|0| Added|
|DatePicker Compat.DateRange- month.chromium.png|0| Added|
|DatePicker Compat.DateRange- week - Dark Mode.chromium.png|0| Added|
|DatePicker Compat.DateRange- week - High Contrast.chromium.png|0| Added|
|DatePicker Compat.DateRange- week.chromium.png|0| Added|
|DatePicker Compat.DateRange- work week - Dark Mode.chromium.png|0| Added|
|DatePicker Compat.DateRange- work week - High Contrast.chromium.png|0| Added|
|DatePicker Compat.DateRange- work week.chromium.png|0| Added|
|DatePicker Compat.Required.chromium.png|0| Added|
|DatePicker Compat.Underlined and required.chromium.png|0| Added|
|DatePicker Compat.Underlined.chromium.png|0| Added|
|DatePicker Compat.With label.chromium.png|0| Added|
|DatePicker Compat.allowTextInput.chromium.png|0| Added|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|0| Added|
|DatePicker Compat.default - Dark Mode.rest.chromium.png|0| Added|
|DatePicker Compat.default - High Contrast.opened.chromium.png|0| Added|
|DatePicker Compat.default - High Contrast.rest.chromium.png|0| Added|
|DatePicker Compat.default.opened.chromium.png|0| Added|
|DatePicker Compat.default.rest.chromium.png|0| Added|
|DatePicker Compat.marked dates - Dark Mode.chromium.png|0| Added|
|DatePicker Compat.marked dates - High Contrast.chromium.png|0| Added|
|DatePicker Compat.marked dates.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>